test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs

```diff
@@ -108,10 +107,7 @@ public void It_migrates_old_dotnet_new_web_without_tools_with_outputs_containing
             var globalDirectory = Path.Combine(projectDirectory, "..");  
             var projectJsonFile = Path.Combine(projectDirectory, "project.json");  
               
-            WriteGlobalJson(globalDirectory);  
-            var projectJson = JObject.Parse(File.ReadAllText(projectJsonFile));  
-            projectJson.Remove("tools");  
-            File.WriteAllText(projectJsonFile, projectJson.ToString());  
+            WriteGlobalJson(globalDirectory);
```

Comment:
No. And I don't think we should. What we talked about and I will take care of in another PR, is make sure that none of the templates will have a cyclic dependency back in the CLI.